projects/components/src/data-exporter/csv-exporter.service.ts

```diff
@@ -44,19 +53,28 @@ export class CsvExporterService {
     }
 }
 
+const SPECIAL_CHAR = new RegExp('^\\+|^-|^=|^@');
```

Comment:
Use literals instead of constructors whenever possible, that way you don't need to double escape.

This could be:

```ts
// Dash has to be the first one in the character sequence, otherwise it would be treated as a range operator. 
// If you think this is too tricky, you could escape it as \-
const LEADING_CONTROL_CHAR = /^[-+=@]/;
```